src/Repository/SkillRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\ORM\EntityRepository;
+
+class SkillRepository extends EntityRepository
+{
+    /**
+     * Finds all availebles skills for autocomplete that connected user has not yet chosen.
+     */
+    public function getSkillsForAutocomplete(string $term, Adherent $user)
+    {
+        $qbUserSkills = $this
+            ->createQueryBuilder('us')
+            ->select('us.name')
+            ->leftJoin('us.summary', 'cv')
+            ->andWhere('cv.member = :user')
```

Comment:
This looks weird to me, I don't get why we need to first select user's skills to then exclude them from all.
I guess we should just query all the tags that should be distinct by added a group by for the slug, then the column name only should be selected so we should use `$query->getArrayResult()` to make the query lighter.